HomeWork 15(salesforce)

MaryManzhos opened the pull request:
add annotations testNG
add Create Account
add Create Contact

MaryManzhos committed:
add Patterns(Page Object, fluent/chain of invocations)
add annotations testNG
add Create Account
add Create Contact

dzmitryrak commented on src/test/java/dataTest/TestData.java:
```diff
@@ -0,0 +1,7 @@
+package dataTest;
```
Переименовать пакет в data

dzmitryrak commented on src/test/java/pages/Accounts/NewAccountModal.java:
```diff
@@ -0,0 +1,51 @@
+        super(driver);
+    }
+
+    public NewAccountModal isPageOpen() {
```
isPageOpened

dzmitryrak commented on src/test/java/pages/Contacts/ContactListPage.java:
```diff
@@ -0,0 +1,30 @@
+import pages.BasePage;
+
+public class ContactListPage extends BasePage {
+    public static final String URL = "https://d2y0000004ceweay.lightning.force.com/lightning/o/Contact/list?filterName=Recent";
```
URL запихнуть тоже в TestData

dzmitryrak commented on src/test/java/pages/Contacts/ContactListPage.java:
```diff
@@ -0,0 +1,30 @@
+        super(driver);
+    }
+
+    public ContactListPage isPageOpen() {
```
isPageOpened

dzmitryrak commented on src/test/java/pages/HomePage.java:
```diff
@@ -0,0 +1,18 @@
+        super(driver);
+    }
+
+    public HomePage isPageOpen() {
```
isPageOpened

dzmitryrak commented on src/test/java/tests/BaseTest.java:
```diff
@@ -0,0 +1,43 @@
+    NewContactModal newContactModal;
+
+    @BeforeClass
+    public void SetUp() {
```
setUp

dzmitryrak commented on src/test/java/tests/RetryAnalyzer.java:
```diff
@@ -0,0 +1,27 @@
+
+public class RetryAnalyzer implements IRetryAnalyzer {
+
+    private static final int MAX_RETRY = 5;
```
3

MaryManzhos committed:
add fields from AccountDetails
set value into model AccountFields

MaryManzhos committed:
add Delete Added Account

MaryManzhos committed:
add Allure reporting

MaryManzhos committed:
add Allure reporting
add attachments

MaryManzhos committed:
rename package dataTest to data
rename method isPageOpen to isPageOpened
rename method Setup to setUp
change value of MAX_RETRY
add retryAnalyzer to test accountShouldBeCreated

MaryManzhos committed:
add check address

MaryManzhos committed:
reformat code

MaryManzhos committed:
add validate added Contact
update ContactTest
add Utils
add new Modals: ContextMenu, DeleteModal, NotificationModal

MaryManzhos committed:
refactoring class Utils

MaryManzhos committed:
reformat code

MaryManzhos committed:
reformat code
add steps
